content/en/continuous_testing/cicd_integrations/configuration.md

```diff
@@ -85,11 +87,20 @@ In the global configuration file, you can configure the following options:
 `datadogSite`
 : The Datadog instance to which request is sent. The default is `datadoghq.com`. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.
 
+`failOnCriticalErrors`
+: A boolean flag that will fail the CI job if no tests were triggered, or results couldn't be fetched from Datadog. The default is set to `false`.
+
+`failOnMissingTests`
+: A boolean flag that will fail the CI job if at least one test is missing within a run (for example, if it has been removed/deleted). The default is set to `false`.
+
+`failOnTimeout`
+: A boolean flag that will fail the CI job if at least one test exceees the default test timeout. The default is set to `true`.
+
```

Comment:
We can add `pollingTimeout` in here as well if it helps making it more evident (we support both `config.pollingTimeout` & `config.global.pollingTimeout`)